docs/users/user_management.md

```diff
@@ -23,6 +23,36 @@ You can [customize it according to your needs](add_login_form.md#customize-login
 The validity of the password recovery token can be set using the `ibexa.system.<siteaccess>.security.token_interval_spec` parameter.
 By default, it is set to `PT1H` (one hour).
 
+### Revoking passwords
+
+In case of a security situation such as a data leakage, you may need to force users to change their passwords.
+You can do it with the help of the `ibexa:user:expire-password`,
+which revokes the passwords for specific users, User Groups or user users belonging to the chosen Content Type.
+
+To select which users to revoke passwords for, use one of the following options with the command:
+
+- `--user-id|-u` - the ID of the user. Accepts multiple user IDs
+- `--user-group-id|-ug` - the ID of the User Group. Accepts multiple group IDs
+- `--user-content-type-identifier|-ct` - the identifier of the user Content Type. Accepts multiple Content Types
+
+You can use the following additional options with the command:
+
+- `--force|-f` - commits the change, otherwise the command only performs a dry run
+- `--iteration-count|-c` - defines how many users are fetched at once. Lowering this value helps with memory issues
+- `--password-ttl|-t` - number of days after which new passwords expire. Used when you update passwords for the whole user Content Type
```

Comment:
> Used when you update passwords for the whole user Content Type

This is not correct. This value is used when user content type does not use password expiration and the command has to enable it, then it uses `--password-ttl` value as "Days before password expires" setting in ezuser fieldtype definition.